pkg/interfaces/contracts/vault/VaultTypes.sol

```diff
@@ -162,6 +166,17 @@ struct TokenInfo {
     bool paysYieldFees;
 }
 
+/**
+ * @dev Data structure used to represent the current pool state in memory
```

Comment:
```suggestion
 * @notice Data structure used to represent the current pool state in memory
```

Since this explains the struct itself, shouldn't it be `@notice`? I understand that `@dev` is used when the dev must pay attention to something in the implementation.

Reply:
Yes - we'd been inconsistent about this. I changed them all to notice, splitting long comments into notice/dev. Good catch!